### ISSUE TITLE
Cache module - micrometer and reactor support

### ISSUE THREAD

**Issue by mkopylec** [resilience4j/resilience4j#513]
Hi, are there any plans to implement it?

**Comment by RobWin** [comment]
No, not yet. Actually I don't know how many people use our `Cache` module.
Reactor has a cache operator in the reactor-addons repo -> https://github.com/reactor/reactor-addons/tree/master/reactor-extra/src/main/java/reactor/cache. But unfortunately no micrometer integration.

Would you like to create a PR?
